src/git.rs

```diff
@@ -153,7 +153,7 @@ impl<'git, 'io, 'sess: 'io, 'ctx: 'sess> Git<'io, 'sess, 'ctx> {
 
                                 // Parse the ref. This is needed since the ref for an annotated
                                 // tag points to the hash of the tag itself, rather than the
-                                // underlying commit. By callign `git rev-parse` with the ref
+                                // underlying commit. By calling `git rev-parse` with the ref
```

Comment:
Very strictly speaking, this typo fix has nothing to do with the "Update links in README" PR. But since it is part of a separate commit, I would not insist but suggest to FF-merge this branch.